src/main/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetails.java

```diff
@@ -1224,11 +1262,13 @@ public String getCsvForAllSubQuestions() {
             this.responseTotalIndex = numOfRubricChoices;
 
             calculateResponseFrequency();
-            calculatePercentageFrequencyAndAverage();
+            calculatePercentageFrequencyAndAverage(false);
```

Comment:
I feel like this is a weird interaction. Is it better if the function returns the value and we set it from outside?

Or since this code is only ever used here, why don't we set both `percentageFrequencyAndAverageExcludingSelf` and `percentageFrequencyAndAverage` in 1 call? Which do you think is cleaner

Reply:
Doing in 1 call would result in a lot of code duplication inside the method, which I have tried to reduce as much as I can.

Reply:
:+1: . I see the effort in reducing code repetition such as using temp, was just thinking that this interaction of calling it twice is very weird.